typescript/sdk/src/token/deploy.ts

```diff
@@ -64,11 +66,16 @@ abstract class TokenDeployer<
     }
   }
 
-  async initializeArgs(_: ChainName, config: TokenRouterConfig): Promise<any> {
+  async initializeArgs(
+    chain: ChainName,
+    config: TokenRouterConfig,
+  ): Promise<any> {
+    const signer = await this.multiProvider.getSigner(chain).getAddress();
```

Comment:
does this raise or lower determinism :P

Reply:
My read is that this is part of the API of `RouterDeployer`

Reply:
I will need to add transferOwnership() in my WarpModule #3678 because i use `deployContracts()` directly